help/en/package/jmri/jmrit/dispatcher/NewTrain.shtml

```diff
@@ -163,6 +164,24 @@
           Block identifier to identify the destination Block uniquely.</p>
         </li>
 
+        <li>
+          <a id="trainDetectionOptions"></a> Train Detection Options <span class="since">since 5.7.5</span>
+          <p> This settings effects the ability of auto trains to stop see autotrains for details</p>
+          <ul>
+            <li><strong>Entire Train</strong> The entire train can be detected, no blocks between head and
```

Comment:
It seems to me that the word "unoccupied" is missing.

no unoccupied blocks between